features/Insights_Advisor_recommendations_filtering.feature

```diff
@@ -0,0 +1,657 @@
+          | Documentation            | no                     |
+     When user expand "Advisor" top level item
+     Then the menu should be expanded under "Advisor" top level item
+      And following new items should be displayed in the sub-menu on the left side
+          | Expanded menu item  | Required for this test |
+          | Clusters            | no                     |
+          | Recommendations     | yes                    |
+     When user select "Recommendations" menu item from this sub-menu
+     Then an "Advisor recommendations" page should be displayed right of the left menu bar
+      And widget with filter settings should be displayed
+      And table with several columns should be displayed
+          | Column name |
+          | Name        |
+          | Added       |
+          | Total risk  |
+          | Clusters    |
+      And that table should contain exactly 1 row
+          | Name        | Added       | Total risk | Clusters |
+          | Bug12345    | 10 days ago | Important  | 1        |
+      And one filter needs to be preselenced
+          | Filter name       | Tag         |
+          | Clusters impacted | 1 or more   |
+     When user clicks on "x" icon displayed on "1 or more" filter tag
+     Then table with advisor recommendation should contain recommendations names with 0 clusters
+          | Name        | Added       | Total risk | Clusters |
+          | Abc12345    | 10 days ago | Important  | 0        |
+          | Bug12345    | 10 days ago | Important  | 1        |
+          | Xyz12345    | 10 days ago | Moderate   | 0        |
+      And filter widget should consists of filter names pull-down menu and filter search input field
+     When user selects "Clusters impacted" in filter names pull-down menu
+      And user select "1 or more" from the rigth pull-down menu
+     Then table with several columns should be displayed
+          | Column name |
+          | Name        |
+          | Added       |
+          | Total risk  |
+          | Clusters    |
+      And that table should contain exactly 1 row
+          | Name        | Added       | Total risk | Clusters |
+          | Bug12345    | 10 days ago | Important  | 1        |
+      And one filter needs to be preselenced
+          | Filter name       | Tag       |
+          | Clusters impacted | 1 or more |
+     When user clicks on "x" icon displayed on "1 or more" filter tag
+     Then table with several columns should be displayed
+          | Column name |
+          | Name        |
+          | Added       |
+          | Total risk  |
+          | Clusters    |
+      And that table should contain three rows
+          | Name        | Added       | Total risk | Clusters |
+          | Abc12345    | 10 days ago | Important  | 0        |
+          | Bug12345    | 10 days ago | Important  | 1        |
+          | Xyz12345    | 10 days ago | Moderate   | 0        |
+
```

Comment:
Why not add "user select "None" from the rigth pull-down menu" case (quick copy-paste, more coverage)

Reply:
added

Reply:
added too :)